src/components/FinishAccount/FinishAccount.jsx

```diff
@@ -25,42 +35,166 @@ const FinishAccount = ({ inviteId, data }) => {
       setErrorMessage(err.message);
     }
   };
+  */
+
+  const passwordRulesTooltipText =
+    '\u2022 Must be at least 8 characters long' +
+    '<br />' +
+    '\u2022 Include at least one uppercase and lowercase letter' +
+    '<br />' +
+    '\u2022 Include at least 1 number' +
+    '<br />' +
+    '\u2022 Include one of these set of special characters [@$!%*?&]';
+
+  const [newPasswordShown, setNewPasswordShown] = useState(false);
+  const toggleNewPasswordVisibility = () => {
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'toggleNewPasswordVisibility' is assigned a value but never used.